its/plugin/projects/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.sonarsource.it.projects</groupId>
```

Comment:
Being on the open source side of SonarSource, I think we should rather use something such as `org.sonarsource.java.it.projects` here, rather than an `com.sonarsource`